src/main/java/seedu/duke/card/Subject.java

```diff
@@ -9,6 +9,18 @@
     private List<Topic> topics;
     private TaskList tasks;
 
+    public Subject(String title){
+        this.title = title;
+    }
+
+    public void printSubject(SubjectList subjectList) {
+        System.out.println("____________________________________________________________\n"
+                + "Got it. I've added this task:\n  " + this + "\n"
+                + "Now you have " + subjectList.getList().size() + (subjectList.getList().size() == 1
+                ? " task in the list.\n" : " tasks in the list.\n")
+                + "____________________________________________________________");
+    }
```

Comment:
I think this printSubject function should be implemented in Ui class instead of here right?